pyproject.toml

```diff
@@ -150,8 +150,12 @@ exclude = [
 ]
 
 [tool.pytest.ini_options]
-python_files = "test_*.py"
-testpaths = "scanpy/tests/"
```

Comment:
This prevented tests being found in `build`, what do we get from removing it?

Reply:
Simplicity. `build` is no longer a thing since we got rid of setuptools.

But we could replace those two lines with `testpaths = "scanpy"` to mirror how we do things in anndata, and enable doctests in the same way.